cmd/csaf_checker/roliecheck.go

```diff
@@ -18,13 +19,33 @@ import (
 	"github.com/csaf-poc/csaf_distribution/v2/util"
 )
 
-// rolieLabelChecker helps to check id advisories in ROLIE feeds
-// are in there right TLP color.
-type rolieLabelChecker struct {
+// identifier consist of document/tracking/id and document/publisher/namespace,
+// which in sum are unique for each csaf document and the name of a csaf document
+type identifier struct {
+	id        string
+	namespace string
+}
+
+// String implements fmt.Stringer
+func (id identifier) String() string {
+	return "(" + id.namespace + ", " + id.id + ")"
```

Comment:
Nitpick: I prefer `fmt.Sprintf()`. But I see it as a matter of taste

Reply:
Not in these string concat cases!